### PR TITLE
Simplified readme + animated gif

### PR DESCRIPTION
**Pull request by auchenberg-stripe**
### Reviewers
r? @
cc @stripe/dev-platform

 ### Summary
Simplified readme as sections have been moved off to the wiki + animated demo gif

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,430 +1,100 @@
...
-Make sure `golangci-lint` is installed: `brew install golangci/tap/golangci-lint`
-
-### Tests
-
-You can run tests with:
-
-```sh
-$ make test
-```
+Got feedback for us? Please don't hersitate to tell us on [feedback](https://stri.pe/cli-feedback).
```
Comment (ob-stripe) [inline README.md]:
```suggestion
Got feedback for us? Please don't hesitate to tell us on [feedback](https://stri.pe/cli-feedback).
```

File: README.md [hunk 2]
```diff
@@ -1,430 +1,100 @@
...
-```sh
-$ sudo apt-get update
-```
+**Feature highlights**:
 
-4. Install the CLI:
+- Securely test webhooks without relying on 3rd party software.
+- Trigger webhook events for easy testing
+- Tail your API request logs in real-time
+- Manage resources by interacting directly with the API to `post`, `get`, `update` or `delete` any Stripe object
```
Comment (ob-stripe) [inline README.md]:
```suggestion
- Manage resources by interacting directly with the API to `create`, `retrieve`, `update` or `delete` any Stripe object
```

File: README.md [hunk 3]
```diff
@@ -1,430 +1,100 @@
...
-    * [config](#config)
-    * [open](#open)
-  * [Developing the Stripe CLI](#developing-the-stripe-cli)
-    * [Installation](#installation-1)
-    * [Linting](#linting)
-    * [Tests](#tests)
-    * [Releasing](#releasing)
-  * [License](#license)  
-    
+The Stripe Command-line interface (CLI) makes it easy for you to build, test and manage your Stripe integration right from the terminal by integrating seamlessly with your Stripe account.
```
Comment (tomer-stripe) [inline README.md]: I kind of don't want to do the "command-line interface (CLI)" part since most people will likely know what a CLI is. We don't do that with API for the docs.

Comment (tomer-stripe) [reply README.md]: "build, test, and manage your Stripe integration right from the command line"

File: README.md [hunk 4]
```diff
@@ -1,430 +1,100 @@
...
-
-3. Update the package list:
-
-```sh
-$ sudo apt-get update
-```
+**Feature highlights**:
 
-4. Install the CLI:
+- Securely test webhooks without relying on 3rd party software.
```
Comment (tomer-stripe) [inline README.md]: Remove the `.` to be consistent with the other bullets

File: README.md [hunk 5]
```diff
@@ -1,430 +1,100 @@
...
+- Trigger webhook events for easy testing
+- Tail your API request logs in real-time
+- Manage resources by interacting directly with the API to `create`, `retrieve`, `update` or `delete` any Stripe object
```
Comment (tomer-stripe) [inline README.md]: `create`, `retrieve`, `update` or `delete` is a subset of what the commands do, should we try to explain more?

File: README.md [hunk 6]
```diff
@@ -1,430 +1,100 @@
...
-```sh
-$ sudo apt-get install stripe
-```
+![demo](docs/demo.gif)
 
-_RedHat/CentOS-based distributions:_
+## Installation
 
-1. Add stripe-cli's yum repository to the yum sources list:
+Stripe CLI is available for MacOS, Windows, and Linux for Debian, Ubuntu, RedHat and CentOS.
```
Comment (tomer-stripe) [inline README.md]: Should be `macOS`. We also technically support more than just the 4 versions of Linux here

File: README.md [hunk 7]
```diff
@@ -1,430 +1,100 @@
...
 
-```sh
-$ wget https://bintray.com/stripe/stripe-cli-rpm/rpm -O bintray-stripe-stripe-cli-rpm.repo && sudo mv bintray-stripe-stripe-cli-rpm.repo /etc/yum.repos.d/
-```
+### macOS
 
-2. Update the package list:
+Stripe CLI is avilable on MacOS via the [package manager Homebrew](https://brew.sh/):
```
Comment (tomer-stripe) [inline README.md]: * `macOS`.
* "Stripe CLI is avilable on MacOS via [Homebrew](https://brew.sh/):" (people will know that homebrew is a package manager)

File: README.md [hunk 8]
```diff
@@ -1,430 +1,100 @@
...
 
 ```sh
-$ sudo yum update
+brew install stripe/stripe-cli/stripe
```
Comment (tomer-stripe) [inline README.md]: I've generally seen people indicate CLI commands by starting with a `$`. What are your thoughts around that here?

File: README.md [hunk 9]
```diff
@@ -1,430 +1,100 @@
...
-$ sudo yum install stripe
-```
+Please refer to the [installation wiki](wiki/installation#linux) for detailed Linux instructions.
 
 ### Windows
 
-_scoop:_
-
-1. Run:
+Stripe CLI is avilable on Windows via the [package manager Scoop](https://scoop.sh/):
```
Comment (tomer-stripe) [inline README.md]: "via the [Scoop] package manager"

File: README.md (hunk 3 above)
Comment (auchenberg) [reply README.md]: The intent here is to align on the value prop and wording used to describe the Stripe CLI, so it's consistent across blog post, stripe docs, and the readme which serves as the "landing page" for the Stripe CLI.

What's the specific proposal?
`The Stripe CLI makes it easy for you to build, test and manage your Stripe integration right from the terminal by integrating seamlessly with your Stripe account.`

File: README.md (hunk 6 above)
Comment (auchenberg) [reply README.md]: Something like `Stripe CLI is available for macOS, Windows, and Linux for distros like Debian, Ubuntu, RedHat and CentOS.`?

File: README.md (hunk 8 above)
Comment (auchenberg) [reply README.md]: I think formatting wise it looks great, but it break copy/paste, which I assume a lot of folks wants to do.

File: README.md (hunk 4 above)
Comment (auchenberg) [reply README.md]: Resolved.

File: README.md (hunk 7 above)
Comment (auchenberg) [reply README.md]: Resolved.

File: README.md (hunk 9 above)
Comment (auchenberg) [reply README.md]: Resolved.

File: README.md (hunk 5 above)
Comment (auchenberg) [reply README.md]: We can just remove it and keep it concise to `Manage resources by interacting directly with the API`?

File: README.md (hunk 8 above)
Comment (tomer-stripe) [reply README.md]: Ah true! Alright resolving

File: README.md (hunk 3 above)
Comment (tomer-stripe) [reply README.md]: Wonder if we should hold off on this until we figure out how to better position the value proposition around the "manage" side

File: README.md (hunk 5 above)
Comment (tomer-stripe) [reply README.md]: Sounds good!

File: README.md (hunk 6 above)
Comment (tomer-stripe) [reply README.md]: That works! For linux, we should either alphabetize it or put Ubuntu first since I imagine that's more popular than some of the rest.

File: README.md (hunk 3 above)
Comment (auchenberg) [reply README.md]: Addressed

**Comment by auchenberg** [comment]
Feedback incorporated. Please review.

File: README.md [hunk 10]
```diff
@@ -11,13 +11,13 @@ The Stripe CLI helps you build, test, and manage your Stripe integration right f
 - Securely test webhooks without relying on 3rd party software
 - Trigger webhook events for easy testing
 - Tail your API request logs in real-time
-- Manage resources by interacting directly with the API to `create`, `retrieve`, `update` or `delete` any Stripe object
+- Manage resources by interacting directly with the API.
```
Comment (tomer-stripe) [inline README.md]: Period here should be removed

Comment (auchenberg-stripe) [reply README.md]: Fixed.

File: README.md [hunk 11]
```diff
@@ -1,430 +1,100 @@
...
-1. Add stripe-cli's yum repository to the yum sources list:
+Stripe CLI is available for macOS, Windows, and Linux for distros like Ubuntu, Debian, RedHat and CentOS.
 
-```sh
-$ wget https://bintray.com/stripe/stripe-cli-rpm/rpm -O bintray-stripe-stripe-cli-rpm.repo && sudo mv bintray-stripe-stripe-cli-rpm.repo /etc/yum.repos.d/
-```
+### macOS
 
-2. Update the package list:
+Stripe CLI is avilable on macOS via the [Homebrew](https://brew.sh/):
```
Comment (tomer-stripe) [inline README.md]: Just "view Homebrew", no "the"

File: README.md [hunk 12]
```diff
@@ -1,430 +1,100 @@
...
-2. `--format json` has the CLI render the status as a JSON blob for easier grepping and for using with tools like `jq`.
-3. `--poll` will continuously check the status site for changes
-4. `--poll-rate` let's you specify how often to check the status site. The default is once every 60 seconds and this can be modified down to once every 5 seconds.
-5. `--hide-spinner` will hide the spinner that's shown when polling.
-
-### `config`
+## Usage
 
-If you need, you can manually set configuration values for the CLI using the `config` command. The config command supports:
+Installing the CLI globally provides access to the `stripe` command.
```
Comment (tomer-stripe) [inline README.md]: Remove "globally", seems unrelated to the command itself.
